common/src/java/com/zimbra/common/localconfig/LC.java

```diff
@@ -66,6 +66,23 @@
     @Supported
     public static final KnownKey zimbra_tmp_directory = KnownKey.newKey("${zimbra_home}/data/tmp");
 
+    /**
+     * Sets whether the External Email Warning (EEW) feature is enabled.
+     */
+    @Supported
+    public static final KnownKey zimbra_external_email_warning_enabled = KnownKey.newKey(false);
+
+    /**
+     * Sets the base message or warning for External Email Warning (EEW).
+     * 
+     * This string will be used and adapted for both text/plain and text/html parts
+     * in an email. Please do not attempt to use HTML mark up here, it will be
+     * escaped anyways.
+     */
+    @Supported
+    public static final KnownKey zimbra_external_email_warning_message = KnownKey
+            .newKey("This message originated outside of your organization. ");
```

Comment:
Is there supposed to be a space at the end of the default message?

Reply:
I opted for a space at the end of the default message mostly because no additional line breaks are appended for the text/plain version, so the space is some sort of separator there. The text/html version is different story, since it's wrapped between a paragraph tag and that renders a clear separation. Please take a look at these lines:
https://github.com/Zimbra/zm-mailbox/blob/874138ab53cfdc66cefa08975d7af89b652ab6bd/store/src/java/com/zimbra/cs/lmtpserver/ExternalEmailWarning.java#L46-L48

As an alternate (and maybe better looking) approach, I suppose I could drop the ending space at the default message, and then change ` textPlainAdmonition = StringEscapeUtils.escapeHtml(baseAdmonition); ` for ` textPlainAdmonition = StringEscapeUtils.escapeHtml(baseAdmonition) + "\r\n"; `

What do you think? Does that sound good, or do you have any other suggestion that we might want to explore?

Reply:
I uploaded the proposed change, I think it will look better. Therefore I also dropped the space at the end of the default message.